src/io/flutter/module/settings/FlutterCreateAdditionalSettingsFields.java

```diff
@@ -111,6 +119,7 @@ public FlutterCreateAdditionalSettings getAdditionalSettings() {
     return new FlutterCreateAdditionalSettings.Builder()
       .setDescription(!descriptionField.getText().trim().isEmpty() ? descriptionField.getText().trim() : null)
       .setType(projectTypeForm.getType())
+      .setAndroidX(projectTypeForm.getType() != FlutterProjectType.PACKAGE && projectTypeForm.getAndroidxCheckbox().isSelected())
```

Comment:
nit: document why FlutterProjectType.PACKAGE should not use AndroidX as it wasn't obvious to me why that is the case.

Reply:
Done.